src/panels/lovelace/components/hui-generic-entity-row.js

```diff
@@ -98,6 +98,9 @@ class HuiGenericEntityRow extends PolymerElement {
                 datetime="[[_stateObj.last_changed]]"
               ></ha-relative-time>
             </template>
+            <template is="dom-if" if="[[_isStateAttribute(config.secondary_info, _stateObj)]]">
```

Comment:
This is not a good approach. It means that if we introduce future options, we might break existing attribute display.

I am also not sure if this is something that we want to begin with. It seems a bit too far away from the simplicity that we thrive for in Lovelace. Probably better off as a custom card.

Reply:
Hmm OK. I think its illogical to not be able to show attributes. I mean there is already entityid and last change. Which are kind of attributes to the state also.
I was surprised it wasn't supported.

I was going to suggest using '''secondary_info: attribute(status)''' or similar but if it's not something you want..

Reply:
I also think it's illogical not to be able to show attributes. What should secondary info else do? As I found out that this is not working, I was very irritated, too!
Putting this feature in a custom card seems to me like a bad workaround. 
`secondary_info: attribute(status)` sounds good to me 👍 This should not be broken by any changes and is easy to understand for beginner users.

Reply:
The rule is that the left side of the row is for entity data, the right side is to display/control the state.

If an entity row is not showing the correct state, that should be fixed. We should not expect every user to add a workaround.

Reply:
@balloob yes, but what does that have to do with this ?

Reply:
You're suggesting to render state data on the left…

Reply:
Ahh yes, true.. But isn't last-updated kind of state data also as it updates with the state?